.travis.yml

```diff
@@ -1,12 +1,11 @@
-before_install:
-   - sudo apt-get update
-   - sudo apt-get install libicu-dev libmozjs-dev
-before_script: ./bootstrap && ./configure
-script: make distcheck
+sudo: false
 language: erlang
 otp_release:
-   - R15B02
-   - R15B01
-   - R15B
-   - R14B04
-   - R14B03
+   - 19.3
+   - 20.3
+   - 21.3
+   - 22.3
+   - 23.1
+script: >
+  curl https://s3.amazonaws.com/rebar3/rebar3 -o rebar3
```

Comment:
I don't think we need this. `rebar3` should be readily available, out-of-the-box. Did you do it because you want the latest version?

Reply:
Is it available? It didn't use to be (I think.)

Reply:
It should be yes. I've been using it for months already.

Reply:
Fixed in ffb96320.